### ISSUE TITLE
Public key menu unclear and unintuitive (not clear you have to click "add" symbol, no indication about multiple keys)

### ISSUE THREAD

**Issue by mat888** [kadena-io/testnet-faucet#1]
On the "get-coins" -> "create and fund new account" of the testnet faucet, it is unclear that (or will be unclear to many):
a) You must click the add symbol even for a single public key
b) That you can add multiple public keys. The prompt text says "Paste and add your public key."

Most forms online, even the "account name" form on this page, simply ask you fill it in. The "public key" form makes no indication that an extra step is required, it can easily be mistaken for a bug on the website. Further more, the text "2. Paste and add your public key" should probably be changed to "2. Paste and add your public key(s)."

TLDR: Make is so a public key entered into the form, but not "added" via the plus button is enough to "Create and Fund Account." Make it clear that multiple public keys can be used. 

This may seem like a minor nitpick, but considering people new to, and not heavily invested (emotionally, time, financially) into Kadena will use this to check out how the network works first hand, its probably very important users don't get frustrated and give up. It doesn't look good for adoption if it feels complicated or unintuitive.
